### PR TITLE
[G2M] Export new components

### PR DESCRIPTION
**Pull request by YizhiCatherineZhang**
- [x] ImageSlider
- [x] StyleCardContainer
- [x] StyledCheckbox
- [x] StyledRadio
- [x] StyledSwitch

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,3 +6,8 @@ export { default as QuickFilters } from './quick-filters'
 export { default as DynamicButton } from './dynamic-button'
 export { default as DynamicDataTable } from './dynamic-data-table' // deprecated by Table
 export { default as Table } from './table'
+export { default as ImageSlider } from './image-slider'
+export { default as StyleCardContainer } from './styled-card-container'
+export { default as StyledCheckbox } from './styled-checkbox'
+export { default as StyledRadio } from './styled-radio'
+export { default as StyledSwitch } from './styled-switch'
```
Comment (DoParkEQ) [inline src/index.js:13]: could you add export { default as Carousel } from './image-slider/carousel' ?

File: src/index.js [hunk 2]
```diff
@@ -6,3 +6,9 @@ export { default as QuickFilters } from './quick-filters'
...
```
Comment (woozyking) [inline src/index.js:13]: at some point we should drop the prefix like `Dynamic` and `Styled` and just go with `Button`, `CardContainer`, `Checkbox`.

Comment (DoParkEQ) [reply src/index.js:13]: I think we can omit prefix and whenever we encounter situation we mix MUI and our own, we can add prefix to mui components?

Comment (woozyking) [reply src/index.js:13]: > I think we can omit prefix and whenever we encounter situation we mix MUI and our own, we can add prefix to mui components?

preciselaay 🧐

Comment (YizhiCatherineZhang) [reply src/index.js:13]: Do we need to include the name change in this PR? If not let's merge this one and then @DoParkEQ can keep working on the marketplace main page.

Comment (woozyking) [reply src/index.js:13]: no need to include the name change. i'll make a backward compatible way of introducing it later

File: src/index.js [hunk 3]
```diff
@@ -6,3 +6,9 @@ export { default as QuickFilters } from './quick-filters'
 export { default as DynamicButton } from './dynamic-button'
 export { default as DynamicDataTable } from './dynamic-data-table' // deprecated by Table
 export { default as Table } from './table'
+export { default as ImageSlider } from './image-slider'
+export { default as StyleCardContainer } from './styled-card-container'
```
Comment (YizhiCatherineZhang) [inline src/index.js:10]: @woozyking @DoParkEQ I just found I had a typo here, should be `StyledCardContainer`. Totally my bad...

**Pull request merged**